core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -261,6 +261,32 @@ public static boolean isLiteralChain(SqlNode node) {
     }
   }
 
+  /**
+   * Unparses a call to an operator which has special syntax.
+   *
+   * @param operator The operator
+   * @param writer   Writer
+   * @param call     List of 0 or more operands
+   */
+  public static void unparseSpecialSyntax(SqlOperator operator,
```

Comment:
this name is really weird. 
could you find a way to `inline` your implementation in the original place instead of here.

Reply:
`SqlUtil#unparseSpecialSyntax` is used for unparse `SqlSyntax.SPECIAL`. other processing logic also exists in SqlUtil.